export_dagmc_cmd/MyPlugin.cpp

```diff
@@ -10,7 +16,15 @@ MyPlugin::~MyPlugin()
 std::vector<std::string> MyPlugin::get_keys()
 {
   std::vector<std::string> keys;
+#ifdef BUILD_DAGMC_EXPORT
   keys.push_back("DAGMCExportCommand");
+#endif
+#ifdef BUILD_IGEOM_TESTS
+  keys.push_back("iGeom_test");
+#endif
+#ifdef BUILD_MCNP_IMPORT
+  keys.push_back("MCNPImp");
+#endif
 
```

Comment:
So far as I can tell, you'd want entirely different files (called in the examples "MyComp") when building the GUI elements, which create "components" which need to be loaded manually in Trelis.  The MyPlugin bit is focused on the command line capabilities.
I could be wrong, though.